### DISCUSSION TITLE
HELP customization of the htmlreport report

### DISCUSSION THREAD

**Discussion by Edmilsontome** [sepinf-inc/IPED#1659] [Q&A]
Hello Good afternoon everyone, I wonder if there is the possibility of customizing the IPED report by changing the PF logo, and replacing the field (FEDERAL CRIMINAL EXPERT and Criminalistics Protocol)

I'm already frowning.

**Comment by lfcnassif** [comment]
You can change the templates per language in `htmlreport` folder

**Reply by milcent-CVM** [reply]
@lfcnassif, is there any instruction/tutorial on how to do this?
Thanks in advance!

**Reply by lfcnassif** [reply]
I think not...
